msgraph/role_eligibility_schedule_requests_test.go

```diff
@@ -0,0 +1,92 @@
+package msgraph_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-azure-sdk/sdk/odata"
+	"github.com/manicminer/hamilton/internal/test"
+	"github.com/manicminer/hamilton/internal/utils"
+	"github.com/manicminer/hamilton/msgraph"
+)
+
+func TestRoleEligibilityScheduleRequestsClient(t *testing.T) {
+	c := test.NewTest(t)
+	defer c.CancelFunc()
+
+	testUser := testUser_Create(t, c)
+
+	roleEligibilityScheduleRequest := testRoleEligibilityScheduleRequestsClient_Create(t, c, msgraph.UnifiedRoleEligibilityScheduleRequest{
+		Action:           utils.StringPtr(msgraph.UnifiedRoleScheduleRequestActionsAdminAssign),
+		DirectoryScopeId: utils.StringPtr("/"),
+		PrincipalId:      testUser.ID(),
+		// ffd52fa5-98dc-465c-991d-fc073eb59f8f is the template id of the "Attribute Assignment Reader" built-in role
+		RoleDefinitionId: utils.StringPtr("ffd52fa5-98dc-465c-991d-fc073eb59f8f"),
+		ScheduleInfo: &msgraph.RequestSchedule{
+			Expiration: &msgraph.ExpirationPattern{
+				Type: utils.StringPtr(msgraph.ExpirationPatternTypeNoExpiration),
+			},
+		},
+	})
+	testRoleEligibilityScheduleRequestsClient_List(t, c)
+	testRoleEligibilityScheduleRequestsClient_Get(t, c, *roleEligibilityScheduleRequest.ID)
+
+	// Processing of the request action takes a little time internally, so it is necessary to sleep for another action
+	time.Sleep(20 * time.Second)
```

Comment:
The `time.Sleep` function is not working as expected, causing the test to fail. This issue is also happening in my local environment. 
I have confirmed that it works correctly when I insert an output process like `fmt.Println("Debug")` before this operation in my local environment as a trial.

Since I am not familiar with Go, if anyone knows the cause of this issue, please let me know.